app/main.py

```diff
@@ -2,0 +1,29 @@
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> None:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
```

Comment:
Code Style issue: Use interpolation instead of concatenation. It is better to use f-strings for formatting strings. Good example: `return f"{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}"`.